learning-bitmark/quick-start/working-with-bitmarks/README.md

```diff
@@ -14,35 +26,46 @@
 * [Registering Bitmark Certificates using the Bitmark App](issuing-bitmarks.md#registering-bitmark-certificates-using-the-bitmark-app)
 * [Registering Bitmark Certificates using the Bitmark SDK](issuing-bitmarks.md#registering-bitmark-certificates-using-the-bitmark-sdk)
 * [Registering Bitmark Certificates using the Bitmark CLI](issuing-bitmarks.md#registering-bitmark-certificates-using-the-bitmark-cli)
-* [Explore the Bitmark transactions using the Bitmark Registry website](issuing-bitmarks.md#explore-the-bitmark-transactions-using-the-bitmark-registry-website)
+* [Exploring Bitmark transactions using the Bitmark Registry website](issuing-bitmarks.md#exploring-bitmark-transactions-using-the-bitmark-registry-website)
 
 
 ## [Transferring Bitmark Certificates](transferring-bitmarks.md#transferring-bitmark-certificates)
   
 * [Prerequisites](transferring-bitmarks.md#prerequisites)
-* [Transferring bitmarks using Bitmark App](transferring-bitmarks.md#transferring-bitmarks-using-bitmark-app)
-* [Transferring bitmarks using Bitmark SDK](transferring-bitmarks.md#transferring-bitmarks-using-bitmark-sdk)
-* [Transferring bitmarks using Bitmark CLI](transferring-bitmarks.md#transferring-bitmarks-using-bitmark-cli)
-* [Explore the Bitmark transactions using the Bitmark Registry website](transferring-bitmarks.md#explore-the-bitmark-transactions-using-the-bitmark-registry-website)
+* [Transferring Bitmarks using the Bitmark App](transferring-bitmarks.md#transferring-bitmarks-using-the-bitmark-app)
+* [Transferring Bitmarks using the Bitmark SDK](transferring-bitmarks.md#transferring-bitmarks-using-the-bitmark-sdk)
+* [Transferring Bitmarks using the Bitmark CLI](transferring-bitmarks.md#transferring-bitmarks-using-the-bitmark-cli)
+* [Exploring Bitmark transactions using the Bitmark Registry website](transferring-bitmarks.md#exploring-bitmark-transactions-using-the-bitmark-registry-website)
 
```

Comment:
In the Bitmark convention, `Bitmark` is using to refer the company name or the Bitmark System. And `bitmark` is using to refer to a record on the blockchain which is a chain of issue record to the last transfer record.

Reply:
Thanks, I fixed it in the table of contents, and found a few other instances in the text.